src/intro_movie.c

```diff
@@ -0,0 +1,197 @@
+#include "global.h"
+#include "intro_movie.h"
+#include "intro_movie_internal.h"
+#include "constants/sndseq.h"
+#include "bg_window.h"
+#include "brightness.h"
+#include "gf_gfx_planes.h"
+#include "main.h"
+#include "math_util.h"
+#include "sound_02004A44.h"
+#include "sys_task_api.h"
+#include "system.h"
+#include "title_screen.h"
+#include "unk_0200B150.h"
+#include "unk_0200FA24.h"
+#include "unk_020215A0.h"
+#include "unk_02022588.h"
+#include "unk_02023694.h"
+
+BOOL IntroMovie_Init(OVY_MANAGER *man, int *state);
+BOOL IntroMovie_Main(OVY_MANAGER *man, int *state);
+BOOL IntroMovie_Exit(OVY_MANAGER *man, int *state);
+void IntroMovie_SetGraphicsBanks(void);
+void IntroMovie_HandleSpriteUpdates(IntroMovieOvyData *data);
+void IntroMovie_InitSpriteGraphicsHW(IntroMovieOvyData *data);
+void IntroMovie_TeardownSpritesManager(IntroMovieOvyData *data);
+
+const OVY_MGR_TEMPLATE gApplication_IntroMovie = {IntroMovie_Init, IntroMovie_Main, IntroMovie_Exit, FS_OVERLAY_ID_NONE};
+
+static BOOL (*sIntroMovieSceneFuncs[])(IntroMovieOvyData *data, void *a1) = {
+    IntroMovie_Scene1,
+    IntroMovie_Scene2,
+    IntroMovie_Scene3,
+    IntroMovie_Scene4,
+    IntroMovie_Scene5,
+};
+
+HeapID _deadstrip_00(int idx);
+HeapID _deadstrip_00(int idx) {
```

Comment:
rename function to `unused` instead of `deadstrip` imo

Reply:
i disagree with that opinion personally